src/pages/LandingPages/MGCovid19/CovidLandingForm.vue

```diff
@@ -268,21 +268,23 @@ export default {
 
 // Monthly or Onetime Toggle
 ::v-deep .deposit-options-toggle {
-	margin-bottom: 1rem;
+	margin-bottom: 2rem;
 
 	.pill {
 		flex-grow: 1;
 	}
 }
 
+::v-deep label,
+::v-deep input {
+	font-weight: 700;
```

Comment:
This makes the numbers inside the pill-toggles bold which matches Ollie's design better and makes the primary actions on this page stand out a little more.

Was hesitant to set this globally without checking everything as it might make items wrap elsewhere in the site.